### PR TITLE
(fix #4819) Allow ionic login to work in a non-interactive process

### LINKED ISSUE #4819: Allow ionic login to work in a non-interactive process
Allow ionic login to work in a non-interactive process. This could be done using the `--confirm` flag which should "_Turn off interactive prompts_"

### PR DESCRIPTION
**Pull request by dtarnawsky**
Allow ionic login to work in a non-interactive process. This could be done using the --confirm flag which should "Turn off interactive prompts"

### CODE REVIEW

File: packages/@ionic/cli/src/commands/login.ts [hunk 1]
```diff
@@ -170,12 +170,15 @@ If you are having issues logging in, please get in touch with our Support[^suppo
...
-      });
+      let login = true;
+      if (!this.env.flags.confirm) {
+        login = await this.env.prompt({
+          type: 'confirm',
+          name: 'continue',
+          message: 'Open the browser to log in to your Ionic account?',
+          default: true,
+        });
+      }
```
Comment (ltm) [inline packages/@ionic/cli/src/commands/login.ts]: Is this change actually needed? I believe the prompt will use the default value in non-interactive mode.

Comment (dtarnawsky) [reply packages/@ionic/cli/src/commands/login.ts]: Looks like it is not. I've removed the unnecessary check

**Review by ltm — APPROVED** [review]
